app/assets/javascripts/controllers/notes/note_utils.js

```diff
@@ -102,7 +102,7 @@ function stringIsUuid(text) {
   const matches = text.match(
     /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/
   );
-  return matches ? true : false;
+  return !!matches;
```

Comment:
Hmm, I guess this just converts the result to a bool, if it's not already. But a bit hard to read IMO.

Reply:
I agree. Should we change the eslint rules to allow for it?